server/game/cards/09.4-CToH/RegalBearing.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class RegalBearing extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Drop your bid to 1 and then draw cards equal to the difference between honor dials',
+            condition: context => context.game.currentConflict.conflictType === 'political' &&
```

Comment:
`context.game.isDuringConflict('political')` is a much safer way of checking this.  This will have a null reference exception if there is no currentConflict.